Mathlib/Algebra/Star/StarAlgHom.lean

```diff
@@ -779,14 +779,14 @@ theorem symm_mk (f f') (h₁ h₂ h₃ h₄ h₅ h₆) :
 theorem refl_symm : (StarAlgEquiv.refl : A ≃⋆ₐ[R] A).symm = StarAlgEquiv.refl :=
   rfl
 
--- should be a `simp` lemma, but causes a linter timeout
-theorem to_ringEquiv_symm (f : A ≃⋆ₐ[R] B) : (f : A ≃+* B).symm = f.symm :=
-  rfl
 
 @[simp]
-theorem symm_to_ringEquiv (e : A ≃⋆ₐ[R] B) : (e.symm : B ≃+* A) = (e : A ≃+* B).symm :=
+theorem toRingEquiv_symm (e : A ≃⋆ₐ[R] B) : (e.symm : B ≃+* A) = (e : A ≃+* B).symm :=
   rfl
 
+@[deprecated (since := "25-08-2025")] alias to_ringEquiv_symm := toRingEquiv_symm
```

Comment:
```suggestion
@[deprecated "← toRingEquiv_symm" (since := "25-08-2025")]
theorem to_ringEquiv_symm (f : A ≃⋆ₐ[R] B) : (f : A ≃+* B).symm = f.symm :=
  rfl
```
or similar, since this is the reverse direction.